src/sonic-py-common/sonic_py_common/multi_asic.py

```diff
@@ -0,0 +1,300 @@
+import glob
+import os
+
+from natsort import natsorted
+from swsssdk import ConfigDBConnector
+from swsssdk import SonicDBConfig
+from swsssdk import SonicV2Connector
+
+from .device_info import CONTAINER_PLATFORM_PATH
+from .device_info import HOST_DEVICE_PATH
+from .device_info import get_platform
+
+ASIC_NAME_PREFIX = 'asic'
+NAMESPACE_PATH_GLOB = '/run/netns/*'
+ASIC_CONF_FILENAME = 'asic.conf'
+FRONTEND_ASIC_SUB_ROLE = 'FrontEnd'
+BACKEND_ASIC_SUB_ROLE = 'BackEnd'
+EXTERNAL_PORT = 'Ext'
+INTERNAL_PORT = 'Int'
+PORT_CHANNEL_CFG_DB_TABLE = 'PORTCHANNEL'
+PORT_CFG_DB_TABLE = 'PORT'
+BGP_NEIGH_CFG_DB_TABLE = 'BGP_NEIGHBOR'
+NEIGH_DEVICE_METADATA_CFG_DB_TABLE = 'DEVICE_NEIGHBOR_METADATA'
+DEFAULT_NAMESPACE = ''
+PORT_ROLE = 'role'
+
+
+def connect_config_db_for_ns(namespace=DEFAULT_NAMESPACE):
+    '''
+    The function connects to the config DB for a given namespace and
+    returns the handle
+    If no namespace is provided, it will connect to the db in the
+    default namespace.
+    In case of multi ASIC, the default namespace is the database
+    instance running the on the host
+
+    Returns:
+      handle to the config_db for a namespace
+    '''
+    SonicDBConfig.load_sonic_global_db_config()
+    config_db = ConfigDBConnector(namespace=namespace)
+    config_db.connect()
+    return config_db
+
+
+def connect_to_all_dbs_for_ns(namespace=DEFAULT_NAMESPACE):
+    '''
+    The function connects to the DBs for a given namespace and
+    returns the handle
+    If no namespace is provided, it will connect to the db in the
+    default namespace.
+    In case of multi ASIC, the default namespace is the
+    database instance running the on the host
+    In case of single ASIC, the namespace has to be DEFAULT_NAMESPACE
+
+    Returns:
+        handle to all the dbs for a namespaces
+    '''
```

Comment:
The docstrings in these functions use single quotes, while the rest of the functions use double quotes. Please change these to double quotes to be consistent with the other docstrings in this file and the other files in sonic-py-common.